src/pages/index.page.tsx

```diff
@@ -2,7 +2,11 @@ import type { NextPage } from 'next';
 import Head from 'next/head';
 import Image from 'next/image';
 
-import styles from '../styles/Home.module.css';
+import styles from 'styles/Home.module.css';
```

Comment:
Do you have a plan to use an alias with something like `@/styles`?
You might have this in other PR, right?
If you want to have this it will be like this in the `tsconfig.json` file.

```
"paths": {
      "@/styles/*": ["styles/*"],
      ...
}
 ```

Reply:
Added in https://github.com/nimblehq/til-web/pull/17/commits/63b249b26105cff41af14379498f44d92b5dfca2
Then I don't need to care it later 😂

Reply:
There was something wrong on my local machine, then I reverted this commit, I will add it later on a UI task